src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs

```diff
@@ -78,14 +79,165 @@ public override string Name
         }
 
         /// <summary>
-        ///  
+        ///
+        /// </summary>
+        public class ConsoleColorProxy
+        {
+            private EditorServicesPSHostUserInterface _hostUserInterface;
+
+            /// <summary>
+            ///
+            /// </summary>
+            public ConsoleColorProxy(EditorServicesPSHostUserInterface hostUserInterface)
```

Comment:
This should be `internal ConsoleColorProxy(EditorServicesPSHostUserInterface hostUserInterface)` , the empty documentation comment can then be removed as well

Reply:
@SeeminglyScience I can't remove empty documentation, when I do travis tell me that : 

`Missing XML comment for publicly visible type or member`

So I had to add these empty documentation to pass your test.